chainer/link_hooks/weight_standardization.py

```diff
@@ -0,0 +1,75 @@
+from chainer.functions.normalization._standardize import _standardize
+from chainer import link_hook
+
+
+class WeightStandardization(link_hook.LinkHook):
+    """Weight Standardization link hook implementation.
+
+    This hook standardizes a weight by *weight statistics*.
+
+    This is a normalization technique that smooths the landscape like batch
```

Comment:
Could you get this sentence clearer?
It is a bit hard for me to guess what kind of landscape does this technique smooth.